geopyspark/geopycontext.py

```diff
@@ -61,6 +61,10 @@ def tile_layer_methods(self):
     def tile_layer_merge(self):
         return self._jvm.geopyspark.geotrellis.spark.merge.MergeMethodsWrapper
 
+    @property
+    def pyramid_builder(self):
```

Comment:
I did that because that's where all of the other Scala objects are, and also because `GeoPyContext` is essentially the bridge between scala and python.